index.html

```diff
@@ -253,5 +253,81 @@ <h3 class="aboutheading">About me</h3>
       </ul>  
     </section>
 
+<!-- Contact form -->
+
+    <section class="text-container">
+      <h2 class="newprojects-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      <div class="contact-form">
+        <form action="https://formspree.io/f/xknylobk" method="post">
+          <input type="text" name="name" class="input-name" placeholder="Full Name" maxlength="30" required>
+          <input type="email" name="email" class="input-email" placeholder="Email Address" required>
+          <textarea name="message" id="message" maxlength="500" class="input-text-area" placeholder="Enter text here"></textarea>
+          <div class="btn">
+          <button class="see2" type="submit">Get in touch</button>
```

Comment:
- [ ] Please add styling for the other states of the button (`active`, `hover` and `disabled`) as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design: 😁

![image](https://user-images.githubusercontent.com/20567503/177320467-4a9575cb-dd8d-4a86-9a6e-a2d7da578cb7.png)

Reply:
I have implemented this